.env.example

```
@@ -5,3 +5,5 @@ REDIS_PORT=6379
 BACKEND_URL=localhost:3000
 # REDIS_USERNAME=
 # REDIS_PASSWORD=
+# INFLUXDB_TOKEN=your influxdb token goes here
+# INFLUXDB_URL=http://localhost:8086
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)